icons/blocks.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="7" height="7" x="14" y="3" rx="1" />
+  <path d="M10 21V8c0-.6-.4-1-1-1H4a1 1 0 0 0-1 1v12c0 .5.5 1 1 1h12c.5 0 1-.5 1-1v-5a1 1 0 0 0-1-1H3" />
```

Comment:
The corners could use arcs instead of cubic curves:
```suggestion
  <path d="M10 21V8a1 1 0 0 0-1-1H4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1v-5a1 1 0 0 0-1-1H3" />
```